### PR TITLE
PLAT-913 Fix DomDayInput display without label

### PR DESCRIPTION
**Pull request by daniel-klose**
image

### CODE REVIEW

File: platform-dom/src/main/java/com/softicar/platform/dom/elements/time/day/AbstractDomDayPopupButton.java [hunk 1]
```diff
@@ -37,7 +37,11 @@ public AbstractDomDayPopupButton() {
 
 	private void updateLabel() {
 
-		setLabel(IDisplayString.create(showLabel? chooser.getDay().toISOString() : ""));
+		if (showLabel) {
```
Comment (alexander-schmidt) [inline platform-dom/src/main/java/com/softicar/platform/dom/elements/time/day/AbstractDomDayPopupButton.java:40]: I think we can revert the changes to this file. Should not make a difference since the change in DomButton was introduced.

Comment (daniel-klose) [reply platform-dom/src/main/java/com/softicar/platform/dom/elements/time/day/AbstractDomDayPopupButton.java:40]: this change makes this class more readable imho, without having to know that `setLabel("")` removes the label

**Review by oliver-richers — APPROVED** [review]
